module/zfs/zfs_vfsops.c

```diff
@@ -1606,9 +1607,8 @@ zfs_domount(struct super_block *sb, zfs_mnt_t *zm, int silent)
 	sb->s_blocksize = recordsize;
 	sb->s_blocksize_bits = ilog2(recordsize);
 	zfsvfs->z_bdi.ra_pages = 0;
```

Comment:
Do we need to modify sb->s_bdi->ra_pages for 4.12+?

Also, we may want to get rid of z_bdi to save space.